default-recommendations/mutability-predicates-test.rkt

```diff
@@ -0,0 +1,41 @@
+#lang resyntax/test
+
+
+require: resyntax/default-recommendations mutability-predicates
+
+
+header:
+------------------------------
+#lang racket/base
+(require racket/contract/base)
+(define (immutable-hash? x) (and (hash? x) (immutable? x)))
+(define (immutable-string? x) (and (string? x) (immutable? x)))
+(define (immutable-bytes? x) (and (bytes? x) (immutable? x)))
+(define (immutable-vector? x) (and (vector? x) (immutable? x)))
```

Comment:
You shouldn't define these yourself. They're defined in the `racket/mutability` module. Replace this code here with the following:

```suggestion
(require racket/contract/base
         racket/mutability)
```